docs/swarm/services.md

```diff
@@ -135,7 +135,7 @@ Swarm mode lets you network services in a couple of ways:
 
 ### Publish ports externally to the swarm
 
-You publish service ports externally to the swarm using the `--publish
+You publish service ports externally to the swarm using the `--publish-add
```

Comment:
But `--publish-add` is only for `docker service update`, not `docker service create`?